### ISSUE TITLE
Only live TV is working since a few weeks

### ISSUE THREAD

**Issue by roelschroeven** [add-ons/plugin.video.vrt.nu#1045]
Since about two weeks only live TV is working for me (on any other stream I tried I get "VRT MAX fout Controleer log voor meer informatie"). This doesn't seem to be the same as issues #1023 and #1024, since it's not solved by upgrading to v2.5.16.

I have the issue on 3 different installations: 2 existing LibreElec installations on raspberry pi + 1 brand new installation on Windows 10.

As far as I understand the Python traceback the issue seems to be with trying to log in. The same credentials work on the VRT MAX website (I double checked to make sure).

Kodi version: Kodi (19.4 (19.4.0) Git:20220302-e12e66e019). Platform: Windows NT x86 64-bit
VRT MAX addon version: v2.5.16+matrix.1

I added the log file in attachment. If there's anything I can try out to help diagnose the problem (like add some extra logging here and there), please let me know.

[kodi.log](https://github.com/add-ons/plugin.video.vrt.nu/files/9609410/kodi.log)

**Comment by mediaminister** [comment]
Can you test the following package:
Kodi 19 Matrix [plugin.video.vrt.nu-2.5.16+matrix.1-fixes-905d28a.zip](https://github.com/add-ons/plugin.video.vrt.nu/files/9610259/plugin.video.vrt.nu-2.5.16%2Bmatrix.1-fixes-905d28a.zip)
Kodi 18 Leia [plugin.video.vrt.nu-2.5.16-fixes-905d28a.zip](https://github.com/add-ons/plugin.video.vrt.nu/files/9610258/plugin.video.vrt.nu-2.5.16-fixes-905d28a.zip)

**Comment by roelschroeven** [comment]
Yes: plugin.video.vrt.nu-2.5.16+matrix.1-fixes-f3b8c44.zip solves the problem

**Comment by roelschroeven** [comment]
I have to remark though that this specific issue only manifested itself on the Windows installation I talked about. The symptoms at first sight were the same on all systems, but the error message in the logs was not. On the other systems I succeeded in making things work by disabling "Toon Mijn favorieten" in the VRT MAX addon settings (I didn't think that was still needed in v2.5.16? Doesn't really matter to me, I don't use those). Terribly sorry for the confusion; I should have had a better look at the different log files before.
